app/views/document/project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=1.7.1
```

Comment:
Can this be deleted?

Reply:
That's from Bledar's PR, I'll check with him